README_sycl.md

```diff
@@ -114,6 +114,12 @@ Output (example):
 
 2. Build locally:
 
+Note:
+
+- By default, it will build for all binary files. It will take more time. To reduce the time, we recommend to build for **example/main** only.
+
+- By default, mmap is used to read model file. In some cases, it leads to the hang issue. Recommend to use build parameter **--no-mmap** to disable mmap() to skip this issue.
```

Comment:
pretty sure its not a "build" parameter.

Reply:
yes, update rm "build".